src/sidebar/media-embedder.js

```diff
@@ -25,11 +25,47 @@ function iframe(src) {
   return iframe_;
 }
 
+/**
+ * Return a formatted YouTube URL query string containing allowed params
+ * e.g. '?t=5'
+ */
+function youTubeQueryParams(link) {
+  var query;
+  // Whitelisted URL query parameters
+  // See https://developers.google.com/youtube/player_parameters
+  const allowedParams = [
+    'end',
+    'start',
+    't',                // Alias for `start`
+  ];
+  const linkParams = queryString.parse(link.search);
+  const filteredQuery = {};
+  // Filter linkParams for allowed keys and build those entries
+  // into the filteredQuery object
+  Object.keys(linkParams)
+    .filter(key => allowedParams.includes(key))
+    .forEach(key => {
+      if (key === 't') {
+        // `t` is a unique param in the YouTube URL API
+        // It works in URLs but not embeds; those require the `start` param
+        // instead. But `t` is commonly used in URLs, so it's worthwhile to
+        // translate it here.
+        return filteredQuery.start = linkParams[key];
```

Comment:
The return value of `forEach` is not used, so the `return` can be dropped from this statement and the one below.